UM/Math/Color.py

```diff
@@ -67,3 +67,25 @@ def fromARGB(value):
             (value & 0x000000ff) >> 0,
             (value & 0xff000000) >> 24
         )
+
+    ##  Returns a new Color constructed from a 7-character string "#RRGGBB" format.
+    #
+    #   \param value A 7-character string representing a color in "#RRGGBB" format.
+    #   \return A Color constructed from the components of value.
+    @staticmethod
+    def fromRGBString(value):
```

Comment:
Argh! QML supports #aarrggbb (http://doc.qt.io/qt-5/qml-color.html), but CSS is going towards #rrggbbaa (https://drafts.csswg.org/css-color/#hex-notation). I think it makes most sense to opt to support QML over CSS4 here.